src/executors/architect.yaml

```diff
@@ -1,3 +1,3 @@
 docker:
   - entrypoint: /bin/bash
-    image: quay.io/giantswarm/architect:latest
+    image: quay.io/giantswarm/architect:a56ef82a3a6a89aaf4db294bc0745961dd1202a2
```

Comment:
Checking if bumping to Go 1.14 will sort my build problems https://github.com/giantswarm/architect/pull/367